crates/rolldown/tests/rolldown/issues/4904/_config.json

```diff
@@ -1,4 +1,5 @@
 {
   "config": {
+    "preserveEntrySignatures": "strict"
```

Comment:
Now the chunk generation still got optimized when `preserveEntrySignature` is struct